frontend/src/column.tsx

```diff
@@ -20,7 +20,7 @@ export function Column(props: ColumnProps) {
          <Flex className="column_header" onClick={() => setOpen(!open)}>
             <Box p={3} pl={4}>{props.title}</Box>
             <Spacer/>
-            <Box className="pull_count" p={3}>{props.pulls.length}</Box>
+            <Box className="pull_count" p={3}>{countPulls(props.pulls, pullsToShow)}</Box>
```

Comment:
It might make sense to move the `pullsToShow` check up into a
```js
const pulls = props.pulls.filter(p => pullsToShow.has(p));
```
above, and then drop the `show` prop below and use
```suggestion
            <Box className="pull_count" p={3}>{pulls.length}</Box>
```
here.

Reply:
> and then drop the show prop below and use

The `show` prop is pretty critical to performance: 29e1548c8ac5cbeff4e5669aab0bc6ea2eb47587